### DISCUSSION TITLE
Copilot installs in IntelliJ but not PHPStorm.

### DISCUSSION THREAD

**Discussion by mrx** [community/community#6736] [Copilot Conversations]
Installs and works without problem in VSC. Install in IntelliJ. PHPStorm can't find the plugin, even with git account setup. 

![Screenshot from 2021-10-25 15-12-37](https://user-images.githubusercontent.com/111910/138779928-604c6ee4-1670-4175-9004-a11410c88522.png)

**Comment by thexkey** [comment]
have you tried updating PHPStorm to at least 2020.2? the extention only works on that version and higher.

**Reply by jansorg** [reply]
@mrx Actually, you need **2021.2** or later, 2020.2 and 2020.3 are incompatible.

**Reply by mrx** [reply]
Thanks. I updated to 2021.2.3, that fixed the error for the plugin repository but I still can't see the copilot plugin. 

I'm going try clearing my config folder.

**Reply by jansorg** [reply]
@mrx That probably won't help. You could try a restart of PhpStorm to force a refresh of the plugin sources. The GitHub Copilot plugin should be shown at the end of the list in the "Marketplace" tab.

**Reply by mrx** [reply]
Rebooted and it's working. Thanks.

**Comment by mrx** [comment]
My version is 2020.3.2 


![Screenshot from 2021-10-26 06-28-14](https://user-images.githubusercontent.com/111910/138889349-b958d91d-7307-4eb4-a588-1186f218b330.png)

**Reply by thexkey** [reply]
have you tried adding `https://plugins.jetbrains.com/plugins/list?channel=super-early-bird`?

**Reply by mrx** [reply]
yes
